### DISCUSSION TITLE
kitty-diff ignore sub folder or file

### DISCUSSION THREAD

**Discussion by cameljava** [kovidgoyal/kitty#4308] [Q&A]
Hi, 

Just start use kitty, really amazed with the functions of kitty provided. 
Running into a issue, but can't find any clue in the documentation, wondering if anyone can help out?

I am using kitty diff to diff two folders, want to ignore certain subfolder inside, eg .git etc, Any idea how to do it?

Thanks a lot.
Kevin

**Comment by kovidgoyal** [comment]
There is no existing option for it, but it should be quite easy to add,
patches are most welcome!

**Reply by cameljava** [reply]
Thanks Kovid for your quick response. Didn't program in python before, may give a crack when free. Just wondering if any chance you can put this in your backlog or as future feature, someone can pick it up. Could be very useful for users.

**Comment by kovidgoyal** [comment]
On Thu, Dec 02, 2021 at 08:30:38PM -0800, Kevin Lee wrote:
> Thanks Kovid for your quick response. Didn't program in python before, may give a crack when free. Just wondering if any chance you can put this in your backlog or as future feature, someone can pick it up. Could be very useful for users.

It's not something I care about personally, have never needed to ignore
directories while doing a recursive diff, but as I said patches are
welcome.

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/pull/5171
